### ISSUE TITLE
Add-to-cart checkbox still visible when $canItemsAddToCart = false

### ISSUE THREAD

**Issue by thdoan** [magento/magento2#6891]
### Preconditions

1. Magento version: 2.1.1
### Steps to reproduce

1. Edit `vendor\magento\module-catalog\view\frontend\templates\product\list\items.phtml`
2. Set all instances of `$canItemsAddToCart` to `false`
3. Go to a product page (e.g., /caesar-warm-up-pant.html)
### Expected result

1. The add-to-cart checkboxes in Related Products should not be visible.
### Actual result

1. The add-to-cart checkboxes in Related Products are still visible:

![image](https://cloud.githubusercontent.com/assets/4110567/19139579/82c27b22-8bb8-11e6-8281-7f18a2216cf2.png)

**Comment by thdoan** [comment]
Proposed fix:

Line 206, change to:

```
<?php if ($canItemsAddToCart && !$_item->isComposite() && $_item->isSaleable() && $type == 'related'): ?>
```

**Comment by sevos1984** [comment]
Thanks for reporting. Internal ticket MAGETWO-59540

**Comment by magento-engcom-team** [comment]
@thdoan, thank you for your report.
We've created internal ticket(s) MAGETWO-59540 to track progress on the issue.

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-82755

**Comment by okorshenko** [comment]
The issue has been fixed and delivered to 2.2-develop branch. Will be available with 2.2.2 release

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-82756

**Comment by magento-team** [comment]
Hi @thdoan. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1284 by @magento-engcom-team in 2.3-develop branch
Related commit(s):


The fix will be available with the upcoming patch release.
